### PR TITLE
add start menu

### PR DESCRIPTION
**Pull request by Elean1402**

### CODE REVIEW

File: src/commonMain/kotlin/scene/MenuScene.kt [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
     override suspend fun Container.sceneInit() {
-        println("Level1")
-        sceneContainer.changeTo<Level1>()
+
+
+        val start = RoundRect(300.0, 50.0, 5.0, fill = Colors["#b9aea0"],).xy(500,300 )
+        val starttext = text ("Start", textSize = 34.0, color = RGBA(0,0,0,255)).xy(600,305)
+        val donate = RoundRect(300.0, 50.0, 5.0, fill = Colors["#b9aea0"],).xy(500,360)
+        val donatetext = text ("Donate", textSize = 34.0, color = RGBA(0,0,0,255)).xy(600,365)
+        val eric = text("diese Einnahmen gehen an Eric's neuen Laptop danke!", textSize = 34.0, color = RGBA(255,255,255,0)).xy(300,420)
```
Comment (alineary) [inline src/commonMain/kotlin/scene/MenuScene.kt:21]: Is this supposed to be there? :D

File: src/commonMain/kotlin/scene/MenuScene.kt [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
+import com.soywiz.korge.input.onClick
 
 
 class MenuScene : Scene() {
     override suspend fun Container.sceneInit() {
-        println("Level1")
-        sceneContainer.changeTo<Level1>()
+
+
+        val start = RoundRect(300.0, 50.0, 5.0, fill = Colors["#b9aea0"],).xy(500,300 )
```
Comment (alineary) [inline src/commonMain/kotlin/scene/MenuScene.kt]: Might just be me but I'd remove the whitespace at the end of the line and always put a whitespace behind a comma for readability

Comment (alineary) [reply src/commonMain/kotlin/scene/MenuScene.kt]: same for the following lines with the commas :D

File: src/commonMain/kotlin/scene/MenuScene.kt [hunk 3]
```diff
@@ -1,12 +1,99 @@
...
 
 
 class MenuScene : Scene() {
     override suspend fun Container.sceneInit() {
-        println("Level1")
-        sceneContainer.changeTo<Level1>()
+        SoundPlayer.playBackgroundMusic("sample_music.mp3")
+
+
+        val start = RoundRect(300.0, 50.0, 5.0, fill = Colors["#b9aea0"]).xy(500, 300)
```
Comment (RobinStarkgraff) [inline src/commonMain/kotlin/scene/MenuScene.kt]: If you write roundRect with a small 'r' infront then it automatically assigns this button to the scene and you don't need the "addchild" anymore which will definitely improve readability :)

File: src/commonMain/kotlin/scene/MenuScene.kt [hunk 4]
```diff
@@ -1,12 +1,99 @@
...
+            volumeiiii.fill = Colors["949494"]
+            SoundPlayer.changeVolume(0.60)
+        }
+        volumeiiii.onClick{
+            volume.fill = Colors["00CD00"]
+            volumei.fill = Colors["00CD00"]
+            volumeii.fill = Colors["00CD00"]
+            volumeiii.fill = Colors["00CD00"]
+            volumeiiii.fill = Colors["00CD00"]
+            SoundPlayer.changeVolume(0.75)
```
Comment (RobinStarkgraff) [inline src/commonMain/kotlin/scene/MenuScene.kt]: The max volume should be 1.0 I guess. Only the default should be in the middle maybe

File: src/commonMain/kotlin/scene/MenuScene.kt [hunk 5]
```diff
@@ -1,12 +1,99 @@
...
+            sceneContainer.changeTo<Level1>()
+        }
+        mute.onClick{
+            volume.fill = Colors["949494"]
+            volumei.fill = Colors["949494"]
+            volumeii.fill = Colors["949494"]
+            volumeiii.fill = Colors["949494"]
+            volumeiiii.fill = Colors["949494"]
+            SoundPlayer.changeVolume(0.0)
+        }
```
Comment (RobinStarkgraff) [inline src/commonMain/kotlin/scene/MenuScene.kt]: You have a resetVolumeColors method which sets all colors to 949494 (which can be a variable ;) ).
After that you have a setColorUntilVolumeiiii method which colors Volumeiiii and calls the setColorUntiVolumeiii method and so on. You can then call any of the setColorUntilVolume methods and they will color all buttons up until the selected one. This saves a lot of manual work and will improve readability.

**Pull request merged**